mithril-aggregator/src/commands/genesis_command.rs

```diff
@@ -229,3 +233,30 @@ impl BootstrapGenesisSubCommand {
         Ok(())
     }
 }
+
+/// Genesis keypair generation command.
+#[derive(Parser, Debug, Clone)]
+pub struct GenerateKeypairGenesisSubCommand {
+    /// Target path for the generated keypair
+    #[clap(long)]
+    target_path: PathBuf,
+}
+
+impl GenerateKeypairGenesisSubCommand {
+    pub async fn execute(
+        &self,
+        root_logger: Logger,
+        _config_builder: ConfigBuilder<DefaultState>,
+    ) -> StdResult<()> {
+        debug!(root_logger, "GENERATE KEYPAIR GENESIS command");
+        println!(
+            "Generating genesis keypair to {}",
+            self.target_path.to_string_lossy()
+        );
+
+        GenesisTools::create_and_save_genesis_keypair(&self.target_path)
+            .with_context(|| "genesis-tools: keypair generation error")?;
+
```

Comment:
Perhaps we could add a success log?

```suggestion
        println!(
            "Genesis keypair generated successfully to {}",
            self.target_path.to_string_lossy()
        );
```

Reply:
I have reformulated the message to keep only one as in other genesis commands.
An eventual error message is displayed after if it fails :+1: